src/main/java/com/ppkwu/ppkwu/services/SampleService.java

```diff
@@ -16,8 +16,32 @@ public String reverse(String stringToReverse, String reverseToString) {
     }
 
     public HashMap recognizeCharacter(String string_To_Recognize) {
```

Comment:
Let's stick to the camelCase naming convention, stringToRecognize will look better.